src/styles/blocks/header.scss

```diff
@@ -0,0 +1,20 @@
+.header {
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  @include font;
```

Comment:
The mixin `font` is used here, but it's not clear what it does. Ensure that the `font` mixin is defined in your mixins file and that it applies the necessary font styles as intended.